Slider/Assets/Scripts/Input/BindingBehaviorComponent.cs

```diff
@@ -0,0 +1,51 @@
+using System;
```

Comment:
Didn't end up using this in the end but it might be useful. It basically lets you put a component on an object to invoke some event when the binding is pressed